nginx/snapcraft.yaml

```diff
@@ -0,0 +1,20 @@
+name: nginx
+version: master
```

Comment:
maybe you should make this a string?
so like this:
```
version: "master"
```
@caldav had this idea when we were making the template, thoughts David?